### ISSUE TITLE
[Known issues] Tracking failures for different branches/main/servicing

### ISSUE THREAD

**Issue by AlitzelMendez** [dotnet/arcade#13459]
Feedback: 
`Tracking failures for different branches /main/ servicing fix?`

We are already running build analysis for other branches although is not easy to see the results, we could improve were this information is showed or where it can be query, we could integrate this to the Landing Page 

### Release Note Category
- [ ] Feature changes/additions 
- [ ] Bug fixes
- [ ] Internal Infrastructure Improvements

**Comment by AlitzelMendez** [comment]
we need more info on this one! I will follow up with the person that provided the feedback

**Comment by AlitzelMendez** [comment]
Hi @SamMonoRT,

Julie shared some feedback with me about Build Analysis/Known issues that I believe came from you. I’m not sure what you are referring to in your feedback. Could you please clarify it for me? Are you talking about how to differentiate known issues by branches? Or are you asking about where to find the Build Analysis?

**Comment by SamMonoRT** [comment]
@AlitzelMendez - My questions around the time this was introduced to us was around 1. How do you differentiate between branches 2. If for some reason we fix an error on main (and not backport to previous version for example 7.0 backport), how will the logic on the build analysis tab work for the 7.0 backport PRs - will they show errors are specific to 8.0 and not 7.0.x to not give a false positive match on the results ?

**Comment by AlitzelMendez** [comment]
Let me try to understand your question. This is what I think you are asking, but please correct me if I’m wrong.

The errors that are shown and matched depend on the specific build. If the build fails with a known issue, the issue is displayed. We don’t show all the existing issues, only the ones that match your specific builds.

For example, suppose you open a known issue that causes an error in a build of the pull request. That known issue issue will be shown when the error occurs. But if you fix the error on main and the error no longer happens on main builds, then the issue will not be shown, on main builds, because there is no match. However, if the issue still appears in other builds (because it hasn’t been fixed on the branch), it will still be shown.

In conclusion we don't differentiate between branches, but we display the issues in a match base, so if the error is not happening and there is not a match is not display, if there is a match it is display.

Please feel free to keep asking if I didn't answer your question.

**Comment by AlitzelMendez** [comment]
I synched offline with Sam and we talked thoroughly about the scenario he was referring, as a quick summary the faced problem was when some helix tests were incorrectly marked as known issues because they were on the same DLL of other test that were actually known issues, this happened some time ago before we introduce the "ExcludeLogs" (exclude helix logs) feature and we talked about it. 

I am closing the issue as this has been clarify but feel free to reopen if there is another question related with this topic.
